index.js

```diff
@@ -92,6 +91,13 @@ function stringToTwoNumbers (num) {
   ]
 }
 
+function fromBase64 (input) {
```

Comment:
Better to call this `base64url` (it's called `fromBase64` in the context of a base64url module)